_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+	This rule checks that the letter spacing can be adjusted by the user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][], for which the `style` attribute [declares][declared][][letter-spacing][] CSS property.
+
+## Expectation 1
+
+For the test target's [letter-spacing][] property, one of the following is true:
+
+- the [declared][] value specified via the `style` attribute does not have the [important flag][]; or
+- the [computed][] value is at least `0.12` times its [computed][] [font-size][].
+
+## Assumptions
+
+If there is a mechanism available on the page by which letter spacing can be adjusted, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
```

Comment:
```suggestion
There is no mechanism available on the page to adjust letter spacing. If there is such a mechanism, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
```
The assumption is the first sentence. The "If" sentence is a fact that is always true, not an assumption.